### DISCUSSION TITLE
course/proxima-book/user-etcd-register

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#133] [General]
# course/proxima-book/user-etcd-register

介绍了如何将用户微服务注册到 etcd 服务注册中心，包括配置文件设置、注册逻辑实现以及服务发现机制的配置。

https://goframe.org/course/proxima-book/user-etcd-register

**Comment by Insua** [comment]
srv.com:2379，这个 srv.com 是自己定义的域名吗？

**Reply by oldme-git** [reply]
这个是`etcd`的访问地址

**Reply by gqcn** [reply]
@oldme-git 我在示例代码仓库中已经修改为了本地地址，避免对阅读者的理解引起混淆，建议更新教程内容。
